src/entries/SampleEntry/SampleEntry.js

```diff
@@ -1,12 +1,148 @@
-import React from "react"
+import React, { useEffect, useRef, useState } from "react"
+import { ClipLoader } from "react-spinners"
+import styled, { keyframes } from "styled-components"
 
-const containerStyle = {
-  width: "800px",
-  height: "600px",
-  backgroundColor: "yellow",
-  margin: "0 auto",
-}
+const appearAnimation = keyframes`
+  from {
+    opacity: 0;
+  }
+
+  to {
+    opacity: 1;
+  }
+`
+
+const Container = styled.div`
+  max-width: 800px;
+  height: 600px;
+  margin: 0 auto;
+`
+
+const Content = styled.div`
+  display: flex;
+  flex-basis: 100%;
+  min-height: 100%;
+  background: #f5f5f5 url(/sample-bg.gif) no-repeat center;
+  background-size: cover;
+  align-items: center;
+  justify-content: center;
+  animation: ${appearAnimation} 0.5s ease;
+`
+
+const Placeholder = styled.div`
+  display: flex;
+  width: 100%;
+  height: 100%;
+  align-items: center;
+  justify-content: center;
+`
+
+const PressContainer = styled.div`
+  font-family: Poiret One;
+  font-size: 3rem;
+  font-weight: 400;
+  margin: 0 auto;
+  color: white;
+  text-shadow: 2px 5px 5px #222222;
+  animation: ${appearAnimation} 0.5s ease;
+
+  @media (min-width: 50rem) {
+    font-size: 5rem;
+  }
+
+  &:hover {
+    cursor: pointer;
+  }
+`
+
+const SampleEntry = () => {
+  const [assetsLoaded, setAssetsLoaded] = useState(0)
+  const [assetsError, setAssetsError] = useState(false)
+  const [chillTime, setChillTime] = useState(false)
+  const audioGainRef = useRef()
+  const audioContextRef = useRef()
+  const completeAssets = assetsLoaded > 1
 
-const SampleEntry = () => <div style={containerStyle}>Hello World</div>
+  const playSound = async () => {
+    const audioContext = new AudioContext()
+    const audioGain = audioContext.createGain()
+    const audioSource = audioContext.createBufferSource()
+    const audioBuffer = await fetch("/sample.mp3")
```

Comment:
Maybe we could create a hook for all the audio-related stuff so participants could focus on creating the actual volume slider.

Reply:
same thoughts as above.